beets/plugins.py

```diff
@@ -488,3 +488,13 @@ def sanitize_choices(choices, choices_all):
             if not (s in seen or seen.add(s)):
                 res.extend(list(others) if s == '*' else [s])
     return res
+
+
+def notify_info_received(msg):
```

Comment:
It took me a very long time to choose a name for this decorator.
The core problem is that passing the yielded value as the info parameter is hardcoded, and that's basically what defines the decorator.
So I feel one way or the other the name should reflect this aspect. I agree that the 'received' part of the name only makes sense in context. To be honest I did not look for other places in the code where this exact decorator could be used.

The other option I thought of, which I think you'd maybe like better given your remark would be `yield_notify_info`. Once you read the code the name makes sense and is very accurate, but I rejected it because:
* In general, I think names that describe what a functions "technically" does are never really good
* If you don't read the code of the function, the name makes very little sense
* I couldn't choose an order and formulation that was concise enough (another problem in using function names which stick to the code, you end up with `do_a_then_b_return_c` which is uglyyyyy)